tests/Microsoft.Identity.Test.iOS.UIAutomation/iOSTests.cs

```diff
@@ -83,11 +83,11 @@ public void RunAllTests()
             var tests = new List<Action>()
             {
                 AcquireTokenTest,
-                AcquireTokenSilentTest,
-                AcquireTokenADFSV3InteractiveFederatedTest,
-                AcquireTokenADFSV3InteractiveNonFederatedTest,
-                AcquireTokenADFSV4InteractiveFederatedTest,
-                AcquireTokenADFSV4InteractiveNonFederatedTest,
+                //AcquireTokenSilentTest,
```

Comment:
dead code?

Reply:
no, these were disabled for app center testing. ill turn them on in another commit.